db/migrate/20171101161031_enable_uuid_extension.rb

```diff
@@ -0,0 +1,5 @@
+class EnableUuidExtension < ActiveRecord::Migration
+  def change
+    enable_extension 'uuid-ossp'
```

Comment:
keep in mind that adding extensions cause breaks on the `db:restore` flow (even using `schema_format = :sql`), usually we solve this by adding it to the `template1` on pg.